uisdk/src/main/java/com/karhoo/uisdk/screen/booking/checkout/component/views/CheckoutView.kt

```diff
@@ -157,28 +158,13 @@ internal class CheckoutView @JvmOverloads constructor(context: Context,
                                     outboundTripId: String?,
                                     bookingMetadata: HashMap<String, String>?,
                                     passengerDetails: PassengerDetails?,
-                                    comments: String?,
-                                    loyaltyInfo: LoyaltyInfo?) {
+                                    comments: String?) {
         loadingButtonCallback.onLoadingComplete()
         bookingCheckoutViewLayout.visibility = View.VISIBLE
         comments?.let {
             bookingRequestCommentsWidget.setBookingOptionalInfo(comments)
         }
-        loyaltyInfo?.let {
-            if (!loyaltyInfo.loyaltyEnabled) {
-                loyaltyView.visibility = GONE
-            }
 
-            loyaltyView.set(LoyaltyViewModel(
```

Comment:
I think we can safely delete the LoyaltyViewModel class. It's not being instantiated anymore from what I see.